float64 goal_lat
float64 goal_lon
---
bool success
